src/derive.rs

```diff
@@ -572,7 +572,7 @@ impl<'a> DerivedModule<'a> {
                 gather: *gather,
                 coordinate: map_expr!(coordinate),
                 array_index: map_expr_opt!(array_index),
-                offset: offset.map(|c| self.import_global_expression(c)),
+                offset: map_expr_opt!(offset),
```

Comment:
What is this code doing and why does this change help?

Reply:
As the person in the git blame, I have no idea. It looks like `naga` just renamed `const_expressions` to `global_expressions` on `Module`s in naga 0.20 so I just copied their renaming.